packages/core/src/shared/lsp/manifestResolver.ts

```diff
@@ -64,9 +64,17 @@ export class ManifestResolver {
         }).getNewETagContent(this.getEtag())
 
         if (!resp.content) {
-            throw new ToolkitError(
-                `New content was not downloaded; fallback to the locally stored "${this.lsName}" manifest`
-            )
+            const msg = `"${this.lsName}" manifest unchanged, skipping download: ${this.manifestURL}`
+            logger.debug(msg)
+
+            const localManifest = await this.getLocalManifest(true).catch(() => undefined)
+            if (localManifest) {
+                localManifest.location = 'remote'
```

Comment:
The common case is that etag didnt change. So that's handled here by re-using `getLocalManifest`. `ManifestResolver` doesn't have a way to tell its caller "I didnt fail but please continue to the next step".